webdriver/tests/classic/navigate_to/file.py

```diff
@@ -10,15 +12,10 @@ def navigate_to(session, url):
 def test_file_protocol(session, target_platform, server_config):
     # tests that the browsing context remains the same
     # when navigated privileged documents
-    path = server_config["doc_root"]
-    if target_platform == "windows":
-        # Convert the path into the format eg. /c:/foo/bar
-        path = "/{}".format(path.replace("\\", "/"))
-    url = u"file://{}".format(path)
+    path = Path(server_config["doc_root"]) / "common" / "blank.html"
+    url = path.as_uri()
 
     response = navigate_to(session, url)
     assert_success(response)
 
-    if session.url.endswith('/'):
-        url += '/'
     assert session.url == url
```

Comment:
Would you mind adding an extra assertion that the `session.url` starts with `file://`? That would make it clearer.